From patchwork Wed Oct 10 22:29:43 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <f4bug@amsat.org>
X-Patchwork-Id: 10635387
Return-Path: 
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 80C7816B1
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 22:31:32 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 6EBB52ABCB
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 22:31:32 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 5F1622ABC8; Wed, 10 Oct 2018 22:31:32 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from lists.gnu.org (lists.gnu.org [208.118.235.17])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mail.wl.linuxfoundation.org (Postfix) with ESMTPS id 04EEB2ABC8
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 22:31:31 +0000 (UTC)
Received: from localhost ([::1]:59409 helo=lists.gnu.org)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>)
	id 1gAN0t-00016E-9d
	for patchwork-qemu-devel@patchwork.kernel.org;
 Wed, 10 Oct 2018 18:31:31 -0400
Received: from eggs.gnu.org ([2001:4830:134:3::10]:36972)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMzL-0008Uv-0z
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 18:29:55 -0400
Received: from Debian-exim by eggs.gnu.org with spam-scanned (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMzJ-0002eV-Mj
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 18:29:54 -0400
Received: from mail-wr1-x442.google.com ([2a00:1450:4864:20::442]:33529)
	by eggs.gnu.org with esmtps (TLS1.0:RSA_AES_128_CBC_SHA1:16)
	(Exim 4.71) (envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMzI-0002dK-2x; Wed, 10 Oct 2018 18:29:52 -0400
Received: by mail-wr1-x442.google.com with SMTP id e4-v6so7466344wrs.0;
	Wed, 10 Oct 2018 15:29:51 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
	h=sender:from:to:cc:subject:date:message-id:in-reply-to:references
	:mime-version:content-transfer-encoding;
	bh=+pvesXanFKnEV/oh0fK1kkjgnm7K/k0QKmfWa8cm2xk=;
	b=AA/7Vz8lzNvM+hR5rzWF3Hzrk08PIGXI/YqH2/YnT0bhYe3QVbQqzCgbQn0DhtVXtG
	oJb6IAuGh21a4sFS4wmRj1L7nHUFnAw+8r5NHUUTCYcXDKsDjiL/5/yvCSE0nbiGPO0d
	2dy8cXJcRV6lTFSobMgWUyEXsjqkmISFQywNH08bphtD3LsI9J7KU4xqS/AajmweZYBn
	TLInz6EPf+IQvChUlGaP5QkKZvs9Ja+zuuUpb/u6n66SW3pP/tfXfGmWEKQWHXp0Rloj
	wi977FzcRchDicHtkaqrVxLJ7cC5EvL/G5vDw2TCsENY3fo4e4L0tIeWkJn30ubv0bUk
	GzRg==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
	d=1e100.net; s=20161025;
	h=x-gm-message-state:sender:from:to:cc:subject:date:message-id
	:in-reply-to:references:mime-version:content-transfer-encoding;
	bh=+pvesXanFKnEV/oh0fK1kkjgnm7K/k0QKmfWa8cm2xk=;
	b=UmbtGS/t2/4epzBqPGAfyjqKVDhNX93JgvLEynfrj0p1mtIaNeMItb5GOZL+DCtARM
	C+TcXQN8JuLRrXnKOxMGeypjxndSEY7sHVXpjTvXO4fWFEBYEi564eXLhS0RA9a33Uq6
	LP/VoiV1gYyFwbOkpTWcTaEGoOeoyqewBnE0X6L9nGXjUQit1F91Ai7uIe0bF1b5fDhs
	K0V+xirkt2SIRdIWsbHhB0ODrVsshHg4K6mRg8/3xcQPDE8eZobJo+Zy7ygNiTtB8k8o
	q8a92dCIhVUI3LkjA+UlAia110DSLRtCWfym5nZCHdjYjcZCIdocR1loKPE1IR3D7kJ6
	s7+A==
X-Gm-Message-State: ABuFfoii+2dO6OfuwZK7IawWebmYclzjGOLUZ5fhAVUztISynJs18tZk
	Ll4LDb+mOVxeXSRpFFRH/Ro=
X-Google-Smtp-Source: 
 ACcGV63To8cfmNx2Ns7lfdWrT/AH2EAOg6Zr/Y52Y/V+7wBGV/wnV1R9KeAmVWN7Fq1hth3/OsU+9Q==
X-Received: by 2002:adf:ae41:: with SMTP id
	u1-v6mr23964516wrd.43.1539210590199;
	Wed, 10 Oct 2018 15:29:50 -0700 (PDT)
Received: from x1.local (26.red-83-32-208.dynamicip.rima-tde.net.
	[83.32.208.26]) by smtp.gmail.com with ESMTPSA id
	c64-v6sm13787447wma.44.2018.10.10.15.29.49
	(version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
	Wed, 10 Oct 2018 15:29:49 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
To: =?utf-8?q?Daniel_P_=2E_Berrang=C3=A9?= <berrange@redhat.com>,
	=?utf-8?q?Reimar_D=C3=B6ffinger?= <Reimar.Doeffinger@gmx.de>,
 Eric Blake <eblake@redhat.com>
Date: Thu, 11 Oct 2018 00:29:43 +0200
Message-Id: <20181010222944.29304-2-f4bug@amsat.org>
X-Mailer: git-send-email 2.19.1
In-Reply-To: <20181010222944.29304-1-f4bug@amsat.org>
References: <20181010222944.29304-1-f4bug@amsat.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
X-detected-operating-system: by eggs.gnu.org: Genre and OS details not
	recognized.
X-Received-From: 2a00:1450:4864:20::442
Subject: [Qemu-devel] =?utf-8?q?=5BPATCH_1/2=5D_mailmap=3A_Fix_Reimar_D?=
	=?utf-8?q?=C3=B6ffinger_name?=
X-BeenThere: qemu-devel@nongnu.org
X-Mailman-Version: 2.1.21
Precedence: list
List-Id: <qemu-devel.nongnu.org>
List-Unsubscribe: <https://lists.nongnu.org/mailman/options/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=unsubscribe>
List-Archive: <http://lists.nongnu.org/archive/html/qemu-devel/>
List-Post: <mailto:qemu-devel@nongnu.org>
List-Help: <mailto:qemu-devel-request@nongnu.org?subject=help>
List-Subscribe: <https://lists.nongnu.org/mailman/listinfo/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=subscribe>
Cc: qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>,
 qemu-devel@nongnu.org
Errors-To: 
 qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org
Sender: "Qemu-devel"
	<qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
X-Virus-Scanned: ClamAV using ClamSMTP

This probably happened when interpreting the utf8 name as latin1.

Fixes dbbaaff6867 and f4e94dfefb6.

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 .mailmap | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/.mailmap b/.mailmap
index 2c2b9b1205..6f2ff22a45 100644
--- a/.mailmap
+++ b/.mailmap
@@ -33,5 +33,6 @@ Justin Terry (VM) <juterry@microsoft.com> Justin Terry (VM) via Qemu-devel <qemu
 
 
 # Also list preferred name forms where people have changed their
-# git author config
+# git author config, or having utf8/latin1 encoding issues.
 Daniel P. Berrangé <berrange@redhat.com>
+Reimar Döffinger <Reimar.Doeffinger@gmx.de>

From patchwork Wed Oct 10 22:29:44 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
X-Patchwork-Submitter: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?=
 <f4bug@amsat.org>
X-Patchwork-Id: 10635389
Return-Path: 
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 55C7E16B1
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 22:31:34 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 3C29D2ABC8
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 22:31:34 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 2CC742ABD1; Wed, 10 Oct 2018 22:31:34 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham version=3.3.1
Received: from lists.gnu.org (lists.gnu.org [208.118.235.17])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mail.wl.linuxfoundation.org (Postfix) with ESMTPS id 75EBA2ABC8
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 22:31:33 +0000 (UTC)
Received: from localhost ([::1]:59410 helo=lists.gnu.org)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>)
	id 1gAN0u-00017N-Ik
	for patchwork-qemu-devel@patchwork.kernel.org;
 Wed, 10 Oct 2018 18:31:32 -0400
Received: from eggs.gnu.org ([2001:4830:134:3::10]:36997)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMzM-0008VV-V1
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 18:29:57 -0400
Received: from Debian-exim by eggs.gnu.org with spam-scanned (Exim 4.71)
	(envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMzL-0002fM-3e
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 18:29:56 -0400
Received: from mail-wr1-x429.google.com ([2a00:1450:4864:20::429]:44162)
	by eggs.gnu.org with esmtps (TLS1.0:RSA_AES_128_CBC_SHA1:16)
	(Exim 4.71) (envelope-from <philippe.mathieu.daude@gmail.com>)
	id 1gAMzK-0002dg-9X; Wed, 10 Oct 2018 18:29:54 -0400
Received: by mail-wr1-x429.google.com with SMTP id 63-v6so7437168wra.11;
	Wed, 10 Oct 2018 15:29:52 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
	h=sender:from:to:cc:subject:date:message-id:in-reply-to:references
	:mime-version:content-transfer-encoding;
	bh=UDbIoAji9yRvj4PmN0OxS0ZP7RJ1XZcNZcBQ7P7BPgk=;
	b=IWvS19ZzhXIi/XZh7SqeDdJ9ksndAcp861Xrs8PW9MCcfT89qxWQKcfk9qY8qXdwNI
	BWKIPL9lgqzfWlUV8+SHcDtKRF7AEQTlKYZJ/OEK1pAYftlZF0coz7bRzUKj7eZbqfx6
	9RvexYkXMT/lmf5pamEnPIz5ognWgmyxjbVuWAr0ATWN0IBLTsnUjUeoJ89eA/ATxqcy
	v6rwl4zFXWs+PmyVSJDDWJDqaeJ/q7OZ189STITB4LqNz1D/1mQCIMAqiGbFVu2cgBgo
	mK+7NX6iyiwjKKji4x3gkC0HTL11STHpFAVxkJ3nzkuYtcuIG4YKBQt0qpeHowwcGqDw
	IxcQ==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
	d=1e100.net; s=20161025;
	h=x-gm-message-state:sender:from:to:cc:subject:date:message-id
	:in-reply-to:references:mime-version:content-transfer-encoding;
	bh=UDbIoAji9yRvj4PmN0OxS0ZP7RJ1XZcNZcBQ7P7BPgk=;
	b=jzkWcsqju5mT+ecmbROoL01yDMqZ4wx9R5LjdylQPf2diy0R4+2G86uys+FlCCgs4g
	t0LuJ1SrBKwaYtSPeqWbTingwvdz2QhyQ9z49E79HuKNuPH86hCazkE7ygcISRHgq8Ej
	7sSIk4g9AYHIYvzaaXmZ1eoAraEigPlP4+x/W3h2R3CjugjFJcc2jeMIyl24C/uggX1K
	DzeXJojZx319Qw+X3CuZQxD7VYnzjP5rBXkXFU3UDIMcy9OYjGtFneyo2qs5JDBKoz3p
	6CkhWVgfJdIyhIIH6QHjO57eLRfEwy3JQ4ft2oB04lFGOy5/MPFiuxFQwJaOIJkb+7XF
	eBHw==
X-Gm-Message-State: ABuFfogRi/impLvLU7L0dVx+Ir9ubAaPz1xyzXMFpB326GgdeqFw0m8O
	9AnLFSruJN1BEf3rNV0tXtc=
X-Google-Smtp-Source: 
 ACcGV61LFZ034NVCFezDFQ2SFXyGEHrE9Z5WlP7SHXUMDGCyzt1x38SbfOTAYWP+hw8uFcpagmcN7Q==
X-Received: by 2002:adf:e808:: with SMTP id
	o8-v6mr23923545wrm.269.1539210591517;
	Wed, 10 Oct 2018 15:29:51 -0700 (PDT)
Received: from x1.local (26.red-83-32-208.dynamicip.rima-tde.net.
	[83.32.208.26]) by smtp.gmail.com with ESMTPSA id
	c64-v6sm13787447wma.44.2018.10.10.15.29.50
	(version=TLS1_2 cipher=ECDHE-RSA-AES128-GCM-SHA256 bits=128/128);
	Wed, 10 Oct 2018 15:29:50 -0700 (PDT)
From: =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>
To: =?utf-8?q?Daniel_P_=2E_Berrang=C3=A9?= <berrange@redhat.com>,
 Eric Blake <eblake@redhat.com>
Date: Thu, 11 Oct 2018 00:29:44 +0200
Message-Id: <20181010222944.29304-3-f4bug@amsat.org>
X-Mailer: git-send-email 2.19.1
In-Reply-To: <20181010222944.29304-1-f4bug@amsat.org>
References: <20181010222944.29304-1-f4bug@amsat.org>
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
X-detected-operating-system: by eggs.gnu.org: Genre and OS details not
	recognized.
X-Received-From: 2a00:1450:4864:20::429
Subject: [Qemu-devel] [PATCH 2/2] mailmap: Split 'prefered emails' from
 'original git import' section
X-BeenThere: qemu-devel@nongnu.org
X-Mailman-Version: 2.1.21
Precedence: list
List-Id: <qemu-devel.nongnu.org>
List-Unsubscribe: <https://lists.nongnu.org/mailman/options/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=unsubscribe>
List-Archive: <http://lists.nongnu.org/archive/html/qemu-devel/>
List-Post: <mailto:qemu-devel@nongnu.org>
List-Help: <mailto:qemu-devel-request@nongnu.org?subject=help>
List-Subscribe: <https://lists.nongnu.org/mailman/listinfo/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=subscribe>
Cc: qemu-trivial@nongnu.org,
 =?utf-8?q?Philippe_Mathieu-Daud=C3=A9?= <f4bug@amsat.org>,
 qemu-devel@nongnu.org
Errors-To: 
 qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org
Sender: "Qemu-devel"
	<qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
X-Virus-Scanned: ClamAV using ClamSMTP

Signed-off-by: Philippe Mathieu-Daudé <f4bug@amsat.org>
---
 .mailmap | 21 +++++++++++++--------
 1 file changed, 13 insertions(+), 8 deletions(-)

diff --git a/.mailmap b/.mailmap
index 6f2ff22a45..8475adf799 100644
--- a/.mailmap
+++ b/.mailmap
@@ -4,26 +4,31 @@
 # into proper addresses so that they are counted properly by git shortlog.
 Andrzej Zaborowski <balrogg@gmail.com> balrog <balrog@c046a42c-6fe2-441c-8c8c-71466251a162>
 Anthony Liguori <anthony@codemonkey.ws> aliguori <aliguori@c046a42c-6fe2-441c-8c8c-71466251a162>
-Anthony Liguori <anthony@codemonkey.ws> Anthony Liguori <aliguori@us.ibm.com>
 Aurelien Jarno <aurelien@aurel32.net> aurel32 <aurel32@c046a42c-6fe2-441c-8c8c-71466251a162>
 Blue Swirl <blauwirbel@gmail.com> blueswir1 <blueswir1@c046a42c-6fe2-441c-8c8c-71466251a162>
 Edgar E. Iglesias <edgar.iglesias@gmail.com> edgar_igl <edgar_igl@c046a42c-6fe2-441c-8c8c-71466251a162>
 Fabrice Bellard <fabrice@bellard.org> bellard <bellard@c046a42c-6fe2-441c-8c8c-71466251a162>
-James Hogan <jhogan@kernel.org> <james.hogan@imgtec.com>
 Jocelyn Mayer <l_indien@magic.fr> j_mayer <j_mayer@c046a42c-6fe2-441c-8c8c-71466251a162>
 Paul Brook <paul@codesourcery.com> pbrook <pbrook@c046a42c-6fe2-441c-8c8c-71466251a162>
-Aleksandar Markovic <amarkovic@wavecomp.com> <aleksandar.markovic@mips.com>
-Aleksandar Markovic <amarkovic@wavecomp.com> <aleksandar.markovic@imgtec.com>
-Paul Burton <pburton@wavecomp.com> <paul.burton@mips.com>
-Paul Burton <pburton@wavecomp.com> <paul.burton@imgtec.com>
-Paul Burton <pburton@wavecomp.com> <paul@archlinuxmips.org>
 Thiemo Seufer <ths@networkno.de> ths <ths@c046a42c-6fe2-441c-8c8c-71466251a162>
 malc <av1474@comtv.ru> malc <malc@c046a42c-6fe2-441c-8c8c-71466251a162>
-
 # There is also a:
 #    (no author) <(no author)@c046a42c-6fe2-441c-8c8c-71466251a162>
 # for the cvs2svn initialization commit e63c3dc74bf.
 
+
+# This section is to track prefered (or last up to date) email for people
+# that uses (or had use) various emails.
+# Example can be people changing from companies, or companies changing their
+# name.
+Aleksandar Markovic <amarkovic@wavecomp.com> <aleksandar.markovic@imgtec.com>
+Anthony Liguori <anthony@codemonkey.ws> Anthony Liguori <aliguori@us.ibm.com>
+James Hogan <jhogan@kernel.org> <james.hogan@imgtec.com>
+Paul Burton <pburton@wavecomp.com> <paul.burton@mips.com>
+Paul Burton <pburton@wavecomp.com> <paul.burton@imgtec.com>
+Paul Burton <pburton@wavecomp.com> <paul@archlinuxmips.org>
+
+
 # Next, translate a few commits where mailman rewrote the From: line due
 # to strict SPF, although we prefer to avoid adding more entries like that.
 Ed Swierk <eswierk@skyportsystems.com> Ed Swierk via Qemu-devel <qemu-devel@nongnu.org>
